pkg/package-server/provider/registry_test.go

```diff
@@ -512,6 +512,22 @@ func TestRegistryProviderGet(t *testing.T) {
 								require.NoError(t, json.Unmarshal([]byte(etcdCSVJSON), &csv))
 								return operators.CreateCSVDescription(&csv, etcdCSVJSON)
 							}(),
+							Entries: []operators.ChannelEntry{
```

Comment:
Given that the entry list for test prometheus and etcd operators is reused throughout, does it make sense to declare them as objects which we can reuse?

Reply:
Done!